lib/src/ovis_util/util.c

```diff
@@ -313,7 +445,11 @@ char *av_value(struct attr_value_list *av_list, const char *name)
 	for (i = 0; i < av_list->count; i++) {
 		if (strcmp(name, av_list->list[i].name))
 			continue;
-		char *str = str_repl_env_vars(av_list->list[i].value);
+		char *tilsub = str_repl_tilde(av_list, av_list->list[i].value, 0);
```

Comment:
Please make the same change in `av_value_at_idx()`.

Reply:
good catch.